.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -13,6 +13,14 @@ object WPComPlugins : Project({
 	params {
 		param("docker_image", "registry.a8c.com/calypso/ci-wpcom:latest")
 		param("build.prefix", "1")
+		checkbox(
```

Comment:
This checkbox lets us skip the diff checking step of the build. That step compares the current build vs. the previous release build to see if the current build has changed anything. That step doesn't work when no release build exists. The checkbox can be used when adding new apps, to manually run builds which skip that diff check. That way, you can get a successful build into the system so that subsequent builds can use it.